### ISSUE TITLE
Plots produced for e.g. standardtest report are pdfs instead of pngs

### ISSUE THREAD

**Issue by fso42** [avaframe/AvaFrame#197]
The files have the extension .png but are actually pdfs...

```
$ file avaInclinedPlane_release1IP_pfd.png                                                          (avaframe) 
avaInclinedPlane_release1IP_pfd.png: PDF document, version 1.4

```

**Comment by fso42** [comment]
Happens when plot Cfg specifies pdf as output -> we need to set the correct fileending

**Comment by fso42** [comment]
Maybe set deflaut in the plotUtilsCfg.ini from pdf to png?

**Comment by fso42** [comment]
Closed with #208
